scripts/check_compliance.py

```diff
@@ -224,7 +248,7 @@ def parse_kconfig(self):
         os.environ['GENERATED_DTS_BOARD_CONF'] = "dummy"
 
         # For multi repo support
-        open(os.path.join(tempfile.gettempdir(), "Kconfig.modules"), 'a').close()
+        get_modules(os.path.join(tempfile.gettempdir(), "Kconfig.modules"))
```

Comment:
How does the root Kconfig.zephyr file find the location of this tempfile?

Reply:
look a few lines up, it is in 

`$(PROJECT_BINARY_DIR)/Kconfig.modules` which translates to 

`os.path.join(tempfile.gettempdir(), "Kconfig.modules")`

Reply:
Oh right. I thought that gettempdir() returned a different one every time.